server/proxy-request.js

```diff
@@ -25,10 +24,39 @@ let passThroughHeaders = function(incomingHeaders) {
 
 
 module.exports = function(url, req) {
-  // support GET or HEAD requests
-  let method = req.method === 'HEAD' ? 'HEAD' : 'GET';
+  // support GET or HEAD or POST requests
+  let methods = ['GET', 'HEAD', 'POST'];
+  let method  = null;
+  if (methods.indexOf(req.method) > -1) {
+    method = req.method;
+  } else {
+    method = 'GET';
+  }
   let externalReqHeaders = passThroughHeaders(req.headers);
 
+  if (method !== 'POST') {
+    externalReqHeaders = u.omit(externalReqHeaders, 'user-agent');
+  }
```

Comment:
Why would we want to change the user agent, and why in POST only? Mind adding some explanation in a comment?